docs_user/modules/proc_migrating-mgr-from-controller-nodes.adoc

```diff
@@ -43,24 +36,23 @@ $ sudo systemctl restart iptables
 label to the target node:
 +
 ----
-ceph orch host label add <target_node> mgr; done
+$ ceph orch host label add <target_node> mgr; done
 ----
 +
 * Replace `<target_node>` with the hostname of the hosts listed in the {Ceph}
-  through the `ceph orch host ls` command
```

Comment:
This text translates to: "Replace `<target_node>` with the hostname of the hosts listed in the Red Hat Ceph Storage through the `ceph orch host ls` command."

That doesn't sound quite right. Should it be something like: “listed in the Red Hat Ceph Storage environment”?